templates/twitter/grafbase/schema.graphql

```diff
@@ -0,0 +1,31 @@
+type User @model {
+  username: String! @unique
+  email: Email! @unique
+  url: URL
```

Comment:
What is url?

Reply:
better named `website` perhaps

Reply:
Let's stick to url since that's what the Twitter API uses.